lntest/btcd.go

```diff
@@ -98,19 +99,28 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		minerAddr: miner,
 	}
 
-	cleanUp := func() {
-		chainBackend.TearDown()
+	cleanUp := func() error {
+		errStr := ""
+		if err := chainBackend.TearDown(); err != nil {
+			errStr += err.Error()
+		}
 
 		// After shutting down the chain backend, we'll make a copy of
 		// the log file before deleting the temporary log dir.
 		logFile := logDir + "/" + netParams.Name + "/btcd.log"
 		err := CopyFile("./output_btcd_chainbackend.log", logFile)
 		if err != nil {
-			fmt.Printf("unable to copy file: %v\n", err)
+			errStr += fmt.Sprintf("unable to copy file: %v\n", err)
```

Comment:
I'd add `\n` if `errStr != ""`

Reply:
Fixed. The newline is added above when the first error appears.